PREP_SYSTEM_FOR_DIETPI.sh

```diff
@@ -1304,7 +1295,7 @@ _EOF_
 			mkdir -p rtl8812au_sparky
 			tar -xvf rtl8812au_sparky.tar -C rtl8812au_sparky
 			chmod +x -R rtl8812au_sparky
-			cd rtl8812au_sparky
+			cd rtl8812au_sparky || exit
```

Comment:
Perhaps a bid dangerous to exit the whole script here without error message directly?

We already error handle the wget download above, which will throw an error and allows to exit or retry via whiptail menu.
If the download succeeds, there should be no reason for the following commands to fail. Okay `tar` could be error handled as well actually, in case of faulty download, corrupted disk and such.
If for some reason you want to assure, that a certain step is done (e.g. critical one, that can easily fail, typically downloads and such), otherwise the script should exit, then I suggest to use our internal error handler `G_RUN_CMD` (as we use with wget above), which gives the user a clear prompt and allows e.g. to manually fix the issue via second terminal/SSH session, retry and in case go on 😃.